specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message",
+          "type": "string"
+        },
+        "target": {
+          "description": "The error target",
+          "type": "string"
+        },
+        "details": {
+          "description": "The error details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model for policy",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "DataPolicyList": {
+      "required": [
+        "values"
+      ],
+      "type": "object",
+      "properties": {
+        "values": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        },
+        "count": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "DataPolicyRoleProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioningState of role",
+          "type": "string"
+        },
+        "roleType": {
+          "description": "The type of role",
+          "type": "string"
+        },
+        "friendlyName": {
+          "description": "The friendly name of role",
+          "type": "string"
+        },
+        "description": {
+          "description": "The description of role",
+          "type": "string"
+        },
+        "cnfCondition": {
+          "$ref": "#/definitions/CnfCondition"
+        },
+        "dnfCondition": {
+          "$ref": "#/definitions/DnfCondition"
+        },
+        "version": {
+          "format": "int64",
+          "description": "The version of role",
+          "type": "integer"
+        }
+      }
+    },
+    "DataPolicyRole": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The Id of role",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of role",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of role",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/DataPolicyRoleProperties"
+        }
+      }
+    },
+    "DataPolicyRoleList": {
+      "required": [
+        "values"
+      ],
+      "type": "object",
+      "properties": {
+        "values": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataPolicyRole"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        },
+        "count": {
+          "format": "int32",
```

Comment:
Is this the total count of entities that one can page through?
If yes, will this be expensive to maintain going forward or expensive to run this on the service?

Reply:
This is the length of the list received in response.